Makefile

```diff
@@ -61,3 +62,21 @@ binaries: ${PREFIX}/bin/registry ${PREFIX}/bin/registry-api-descriptor-template
 clean:
 	@echo "+ $@"
 	@rm -rf "${PREFIX}/bin/registry" "${PREFIX}/bin/registry-api-descriptor-template"
+
+	
+# Use the existing docs build cmds from docker/docker
+# Later, we will move this into an import
+DOCS_MOUNT := $(if $(DOCSDIR),-v $(CURDIR)/$(DOCSDIR):/$(DOCSDIR))
+DOCSPORT := 8000
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
```

Comment:
The following two lines:

```
GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
DOCKER_DOCS_IMAGE := docker-docs$(if $(GIT_BRANCH),:$(GIT_BRANCH))
```

May be replaced with this:

```
DOCKER_DOCS_IMAGE := docker-docs-$(VERSION)
```

The branch name may not be very interesting to keep the image name unique. If the goal is for this to have local branch name (ie "master"), then please disregard.